webapp/src/css/dropdown.less

```diff
@@ -106,6 +107,26 @@
       content: none;
     }
     li {
+      > a[role="menuitem"] {
+        padding: 5px 20px 3px 35px;
+        text-indent: -20px;
+
+        &.indent-1 {
+          padding-left: 50px;
+        }
+
+        &.indent-2 {
+          padding-left: 60px;
+        }
+
+        &.indent-3 {
+          padding-left: 70px;
+        }
+
+        &.indent-4 {
+          padding-left: 80px;
+        }
+      }
```

Comment:
Can we somehow merge this with the indent code around line 250? I think the indents are only used in multidropdowns so we should be able to just define them in one place.

Reply:
In this case I don't see another way other than re-build part of the markup, the extra padding here is needed because the way the text is indented after the first line, because the ☑️ check is not a real check component, it's just a character appended before the text with CSS.

Reply:
Right, but I think the indents are _only_ currently used with multidropdowns so I think you can just change (or remove) the CSS in the other block. Unless I'm missing somewhere where we use indents with a single choice dropdown?

Reply:
Got it, yes I was not sure whether multi level dropdowns are used without multi-select, but look like aren't, so I removed the old indents and kept the new ones just for multi-select.